src/Microsoft.AspNetCore.SystemWebAdapters.FrameworkServices/ProxyHeaderModule.cs

```diff
@@ -42,10 +44,12 @@ public void Init(HttpApplication context)
 
         if (options.UseForwardedHeaders)
         {
+            _reflectionHelper = new ReflectionHelper();
```

Comment:
This will only allow the reflection helper stuff to run when in this code path, but it should be available for both. Can't this be initialized in the constructor?

Reply:
Do we need to run the reflection path when we aren't forwarding headers?  At least from my reading of the code, we don't do need to do anything expect when the host is forwarded, since that's all the reflection helper is setting anywyas

Reply:
Okay I will just move this to the constructor